extensions/shared/blocks/cover/edit.js

```diff
@@ -8,8 +8,9 @@ import { createHigherOrderComponent } from '@wordpress/compose';
 /**
  * Internal dependencies
  */
-import { isUpgradable, isVideoFile } from './utils';
+import { isVideoFile } from './utils';
 import { CoverMediaProvider, JetpackCoverUpgradeNudge } from './components';
+import { isUpgradable } from "../../paid-blocks/utils";
```

Comment:
Could you do another `"` vs `'` pass?

Reply:
Sorry, it's added automatically by my IDE. :-(